proto/cosmos/base/abci/v1beta1/abci.proto

```diff
@@ -48,6 +48,25 @@ message TxResponse {
   repeated tendermint.abci.Event events = 13 [(gogoproto.nullable) = false];
 }
 
+// BlockResponse
+message BlockResponse {
```

Comment:
are we able to use https://github.com/cosmos/cosmos-sdk/blob/3676b2f2e70e54062eada1719ccbb7c2429fc966/proto/tendermint/types/block.proto#L10 to not duplicate types?

Reply:
yeah, thank you for showing me that... I will use that proto